dev_guide/templates.adoc

```diff
@@ -51,41 +51,22 @@ A template describes a set of resources intended to be used together that can be
 
 ====
 
-== Labels
-link:../architecture/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template. 
+== Creating a Configuration From a Template
 
-There is also the ability to add labels in the template from the command line.
+You can create a configuration from a template with either the Management Console or the CLI, after first uploading a template to your OpenShift instance.
```

Comment:
Yes...  osc process can take two types of input:
1) the name of a template that has been uploaded to the openshift template library.
  - osc process templatename
2) the name of a file on the local filesystem
  - osc process -f filename.json

In the web console, you can effectively only do #1 (you have to reference a template that has been uploaded), but when using the CLI you can do #1 or #2.  

Obviously if you have a file on the filesystem you can always upload it first, and then do #1, also.